src/core/core.tooltip.js

```diff
@@ -95,7 +95,8 @@ module.exports = function(Chart) {
 			// Args are: (tooltipItems, data)
 			beforeFooter: helpers.noop,
 			footer: helpers.noop,
-			afterFooter: helpers.noop
+			afterFooter: helpers.noop,
+			textLabelColor: helpers.noop
```

Comment:
yes. i tried that, but the default color at that moment returned to be black. instead of white.